Backpack/src/main/res/layout/view_bpk_calendar_day.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="wrap_content"
+  android:gravity="center_horizontal|top"
+  android:orientation="vertical"
+  android:paddingBottom="24dp">
+
+  <net.skyscanner.backpack.text.BpkText
+    android:id="@+id/bpk_calendar_cell_date"
+    android:layout_width="match_parent"
+    android:layout_height="36dp"
```

Comment:
same here

Reply:
i think this is still hardcoded in the other views

Reply:
Same